src/modules/errorHandling/unhandledError.js

```diff
@@ -1,12 +1,11 @@
-import { getClient, userDb, responseDb, keyDb, factionDb } from '../../util';
+import { factionDb, keyDb, responseDb, userDb } from '../../dbs';
+import { getClient } from '../../util';
 
 process.on('uncaughtException', (err) => {
     console.error('Uncaught Error!!!!\nSomebody better fix this!!!');
     console.error(err);
     getClient()?.destroy();
-    userDb.close();
-    responseDb.close();
-    keyDb.close();
-    factionDb.close();
-    process.exit(1);
+    Promise.all([userDb.close(), responseDb.close(), keyDb.close(), factionDb.close()]).then(
+        process.exit(1)
+    );
```

Comment:
These are not promises

Reply:
In fact we no longer need to call these at all when we update to latest Enmap

Reply:
This is still an issue